### ISSUE TITLE
IoU threshold in IoU Net

### ISSUE THREAD

**Issue by Mulan337287** [vacancy/PreciseRoIPooling#23]
Hi, in your paper, predicted boxes with IoU larger than 0.5 will be fed into IoUNet.
I have a question that did you have a try to set the threshold as 0.0? Though IoU>0.5, we will regard it as positive boxes, but does IoU net have the ability to distinguish boxes with low IoU?
If IoU net only generate IoU score larger than 0.5,  will it affect the performance?

**Comment by vacancy** [comment]
No, definitely if all boxes during training are of IoU>0.5, you can not expect the model to generalize to low-IoU boxes. This threshold is important since using all bounding boxes to train the IoU head (boxes with arbitrary IoU) will affect the performance of the prediction accuracy. (simply because there are much more training examples with larger visual variance)

**Comment by joishbader** [comment]
I have some extra words about the thresholds we've tried at the bottom of  #36.
If you are interested that might be helpful.
